relentless/simulate/hoomd.py

```diff
@@ -401,15 +401,22 @@ class MinimizeEnergy(simulate.SimulationOperation):
         Force convergence criterion.
     max_iterations : int
         Maximum number of iterations to run the minimization.
-    dt : float
-        Maximum step size.
+    options : dict
+        Additional options for energy minimizer.
+
+    Raises
+    ------
+    ValueError
```

Comment:
This should be a `KeyError` (not a `ValueError`) because the option is a key of the dictionary.